src/main/java/gregtech/api/gui/impl/ModularUIGui.java

```diff
@@ -72,6 +72,11 @@ public void handleWidgetUpdate(PacketUIWidgetUpdate packet) {
 
     @Override
     public void drawScreen(int mouseX, int mouseY, float partialTicks) {
+        timeSinceUpdate += partialTicks;
+        if(timeSinceUpdate > 0.3) {
+            this.timeSinceUpdate -= 0.3;
+            modularUI.guiWidgets.values().forEach(Widget::updateScreenOnFrame);
+        }
```

Comment:
why it is 60fps.

```
long now = System.currentTimeMillis();
if (now - last >= 17) {
    last = now;
    modularUI.guiWidgets.values().forEach(Widget::updateScreenOnFrame);
}
```

Reply:
It approximates a third of a tick, making it 60 FPS. Also, didn't you want this to use partialTicks?

Reply:
but idk whats your logic (`timeSinceUpdate += partialTicks;`)
Can you explain it to me? Let's say the current FPS is 120 (video settings). 
Ideally the period for partialsTicks in some case would be like this:
 (1/6=> 2/6=>3/6=>4/6=>5/6=>6/6) hit in one tick, but your logic here will be hit 4 times (3,4,5,6) (80fps). 
another example, also Ideally the period for partialsTicks but different initial case:
 (0/6=>1/6=> 2/6=>3/6=>4/6=>5/6), your logic will be hit 3 times (3,4,5) (60fps).
In fact, partialsTicks are not so ideal, the FPS you simulate can be higher or smaller depending on the FPS of the game, not 60.

imo, using partialsTicks should be:
```
float now = getModularUI().entityPlayer.ticksExisted + partialTicks;
if (now - last >= 0.333) {
    last = now;
    modularUI.guiWidgets.values().forEach(Widget::updateScreenOnFrame);
}
```
If this is my math problem, please ignore this comment. I just scribbled the math in my head, not in detail to calculate the expectation or your logic.